app/models/katello/docker_manifest.rb

```diff
@@ -2,7 +2,7 @@ module Katello
   class DockerManifest < Katello::Model
     include Concerns::PulpDatabaseUnit
 
-    has_one :docker_tag, :dependent => :destroy, :class_name => "Katello::DockerTag"
+    has_many :docker_tags, :dependent => :destroy, :class_name => "Katello::DockerTag"
```

Comment:
Almost there. The sync looked good. Just see if we can get the json fixed and I think its good to ack after..